src/main/java/at/hannibal2/skyhanni/features/misc/limbo/LimboPlaytime.kt

```diff
@@ -0,0 +1,111 @@
+        }
+        event.replaceWith(limboItem)
+    }
+
+    @SubscribeEvent
+    fun onHoverItem(event: LorenzToolTipEvent) {
+        if (!LorenzUtils.inSkyBlock) return
+        if (!event.slot.inventory.displayName.unformattedText.startsWith("Detailed /playtime")) return
+        if (event.slot.slotIndex != 4) return
+        if (config.limboPlaytime == 0) return
+
+        val lore = event.toolTip
+        val hoursArray = lore.filter { hoursRegex.matches(it) }.toMutableList()
+        val minutesArray = lore.filter { minutesRegex.matches(it) }.toMutableList()
+
+        addLimbo(hoursArray, minutesArray)
+        remakeArray(event.toolTip, minutesArray, hoursArray)
+    }
+
+    @SubscribeEvent
+    fun onRenderGUI(event: InventoryOpenEvent) {
+        if (event.inventoryName != "Detailed /playtime") return
+        if (config.limboPlaytime == 0) return
+        val playtime = config.limboPlaytime.seconds
+        val wholeHours = playtime.inWholeHours
+        wholeMinutes = playtime.inWholeMinutes
+        if ((wholeMinutes%60).toInt() == 0) {
+            hoursString = "$wholeHours"
+        } else {
+            val minutes:Float = ((wholeMinutes - wholeHours * 60).toFloat() / 60)
+            hoursString = wholeHours.addSeparators()+minutes.round(1).toString().replace("0", "")
+        }
+    }
+
+    private fun addLimbo(hoursArray: MutableList<String>, minutesArray: MutableList<String>) {
+        if (wholeMinutes >= 60) {
+            val hours = config.limboPlaytime.seconds.inWholeHours
+            val minutes = (config.limboPlaytime.seconds.inWholeMinutes-(hours*60).toFloat()/6).toInt()
+            modifiedArray = hoursArray
+            if (minutes == 0) modifiedArray.add("§b$hours hours §7on Limbo")
+            else modifiedArray.add("§b$hoursString hours §7on Limbo")
+            modifiedArray = modifiedArray.sortedByDescending {
+                it.substringAfter("§b").substringBefore(" hours").toDoubleOrNull()
+            }.toMutableList()
+            setMinutes = false
+        }
+        else {
+            val minutes = config.limboPlaytime.seconds.inWholeMinutes
+            modifiedArray = minutesArray
+            modifiedArray.add("§a$minutes minutes §7on Limbo")
+            modifiedArray = modifiedArray.sortedByDescending {
+                it.substringAfter("§a").substringBefore(" minutes").toDoubleOrNull()
+            }.toMutableList()
+            setMinutes = true
+        }
+    }
+
+    private fun remakeArray(toolTip: MutableList<String>, minutesArray: MutableList<String>, hoursArray: MutableList<String>) {
+        val firstLine = toolTip.first()
+        val totalPlaytime = toolTip.last()
+        toolTip.clear()
+        toolTip.add(firstLine)
+        if (!setMinutes) {
+            toolTip.addAll(modifiedArray)
+            toolTip.addAll(minutesArray)
+        } else {
+            toolTip.addAll(hoursArray)
+            toolTip.addAll(modifiedArray)
+        }
```

Comment:
You could do `toolTip.addAll(modifiedArray)` outside of the if statement

Reply:
how would i do this? the order where they get added matters

Reply:
@CalMWolfs